TestSwitchApi/TestSwitchApi/Repositories/CandidateTestsRepo.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using TestSwitchApi.Models.ApiModels;
+using TestSwitchApi.Models.DataModels;
+using TestSwitchApi.Models.Request;
+
+namespace TestSwitchApi.Repositories
+{
+    public class CandidateTestsRepo : ICandidateTestsRepo
+    {
+        private readonly TestSwitchDbContext _context;
+
+        public CandidateTestsRepo(TestSwitchDbContext context)
+        {
+            _context = context;
+        }
+
+        public IEnumerable<CandidateTestModel> GetSubmissionsById(int id)
```

Comment:
Is `GetSubmissionsByCandidateId` clearer - just because there are few Ids kicking about here! :)

Reply:
GetSubmissionsByCandidateId is much clearer- have renamed the method